worksheet.rb

```diff
@@ -31,4 +31,147 @@
 # - the total amount of money each driver has made
 # - the average rating for each driver
 # - Which driver made the most money?
-# - Which driver has the highest average rating?
\ No newline at end of file
+# - Which driver has the highest average rating?
+
+rides =
```

Comment:
You end up calling your hash of data `rides`. However, when i look at what the keys and values are, I'm not sure that `rides` is the most accurate name for entire hash.

I think that the values of `dr0001`, `dr0002` etc represent an array of `rides`, but I might consider a different name for this entire hash.